.circleci/config.yml

```diff
@@ -11,6 +11,7 @@ jobs:
       - image: circleci/golang:1.11.13
         environment:
           TEST_RESULTS_DIR: *test_results_dir
+          GO111MODULE: "on"
```

Comment:
just for my own knowledge because i still don't understand what this does: what's the impact of turning this on here? like what does it change/do?

Reply:
Line 16 of this config sets the working directory to a folder within the `GOPATH`:

```
/go/src/github.com/terraform-providers/terraform-provider-tfe
```

`GO111MODULE` defaults to `"auto"` which for Go 1.11 and 1.12 means not using go modules when inside the Go path (but changes with 1.13 to using modules while inside the Go path). We are using modules to track our dependencies and have been updating the vendor directory but with this flag set it is not necessary to continue to vendor these dependencies in the vendor folder.

I will vendor the dependency and take this out.